src/main/io/osd.h

```diff
@@ -83,6 +83,7 @@ typedef enum {
     OSD_COMPASS_BAR,
     OSD_ESC_TMP,
     OSD_ESC_RPM,
+    OSD_REMAINING_TIME_ESTIMATE,
```

Comment:
Normally, for a lot of the enums in Betaflight, new entries **have to** go at the bottom, since the elements are referenced by index in MSP, and inserting value will break the existing functionality / make life harder for the person doing the configurator side change.
Since `OSD_RTC_DATETIME` and `OSD_ADJUSTMENT_RANGE` are both brand new we can let this slide, provided the configurator side change is done in a timely fashion.

Reply:
I noticed this during the rebase but wasn't sure if it should be moved... Next time 👍